### PR TITLE
CRDs should be visible at Cluster Level Only

### LINKED ISSUE #1115: CRDs should be visible at Cluster Level Only
As [Sebastian](https://github.com/kyma-project/community/blob/main/manifesto/personas.md#sebastian---tech-lead) and [Mathew](https://github.com/kyma-project/community/blob/main/manifesto/personas.md#matthew---cloud-native-software-engineer) I would expect that I see all CRDs at cluster level ( cluster bound and namespace bound ).

**Acceptance Criteria**

- [x] Follow Project DOD
- [x] All CRDs are visible at Cluster level only.
- [x] Namespace CRD will display a complete list of all instances, specifying the namespace, and deep linking to the namespace
- [x] At Namespace level we display Custom Resources only, the list is cluster by group and the detail page just lists the instances with deep link to instance detail page.
- [x] Display Custom Resources on Cluster level, too.
- [x] Tests are present testing the functionality

**Tasks**

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes NOT proposed in this pull request:

- Handle CR list & CR details in command palette - it's to be done in #1110 

**Related issue(s)**

Closes #1115

### CODE REVIEW

**Review by parostatkiem — DISMISSED** [review]

**Review by dariadomagala — COMMENTED** [review]
Links in the `Related CRDs` don't work.

Can we use 'kind' name in the create button, to also have the uppercase (as we have in the title) 
Screen Shot 2022-03-28 at 08 14 51

**Review by dariadomagala — COMMENTED** [review]
Also, please adjust tests that are failing

**Review by dariadomagala — DISMISSED** [review]

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -757,6 +759,10 @@ custom-resource-definitions:
     not-served: This version is not served using REST APIs.
     served: This version is served using REST APIs.
     storage: Use this version when persisting custom resources to storage.
+custom-resources:
+  description: <0>Custom Resources</0> extend the Kubernetes API. You can define your own Custom Resource by creating a <1>Custom Resource Definition</1> first.
```
Comment (grego952) [inline core/src/i18n/en.yaml:763]:
```suggestion
  description: <0>Custom resource</0> extends the Kubernetes API. You can define your own custom resource by creating a <1>Custom Resource Definition</1> first.
```

**Review by dariadomagala — APPROVED** [review]

**Pull request merged**